src/components/modals/VoteHistoryModal.vue

```diff
@@ -18,21 +18,24 @@
           <td>Nominator</td>
           <td>Nominee</td>
           <td>Type</td>
-          <td>Majority</td>
+          <td>Votes</td>
           <td><font-awesome-icon icon="hand-paper" /> Hand up</td>
+          <td>Passed</td>
         </tr>
       </thead>
       <tbody>
         <tr v-for="(vote, index) in session.voteHistory" :key="index">
           <td>{{ vote.nominator }}</td>
           <td>{{ vote.nominee }}</td>
           <td>{{ vote.type }}</td>
-          <td>{{ vote.majority }}</td>
+          <td><strong>{{ vote.votes.length }}</strong> / {{ vote.majority }}</td>
           <td>
-            {{ vote.votes.length }}
             <font-awesome-icon icon="user-friends" />
             {{ vote.votes.join(", ") }}
           </td>
+          <td>
+            <font-awesome-icon icon="check-square" v-if="vote.votes.length >= vote.majority" />
```

Comment:
...and add it here in parenthesis.

```suggestion
            <template v-if="vote.votes.length >= vote.majority">
              <font-awesome-icon icon="check-square" /> ({{ vote.majority }})
            </template>
```

Reply:
Ah, I agree, that seems to be a more sensible spot to put it.